packages/python-pulpcore/python-pulpcore.spec

```diff
@@ -130,6 +130,8 @@ sed -i '/cryptography/ s/~=.*//' requirements.txt
 set -ex
 %py3_build
 %{?scl:EOF}
+#Adding a sed to change redis on requirements, from ~= to >=
+sed -i '/redis~=3.4.0/c\redis>=3.4.0' requirements.txt
```

Comment:
I saw a redis related repoclosure failure. Is this the cause?

Reply:
Yeah, they changed from >= to ~=, and this broke the repoclosure